sympy/solvers/tests/test_solveset.py

```diff
@@ -1923,18 +1923,33 @@ def test_uselogcombine_2():
     assert solveset_real(eq, x) == EmptySet()
 
 
-def test_is_logarithmic():
-    assert _is_logarithmic(log(x), x) is True
-    assert _is_logarithmic(log(x)*log(y), x) is True
-    assert _is_logarithmic(log(x)**2, x) is True
-    assert _is_logarithmic(log(x**y) - y*log(x), x) is True
-    assert _is_logarithmic(sin(log(x)), x) is False
-    assert _is_logarithmic(log(x) + log(x + 3), y) is False
-    assert _is_logarithmic(x + y, x) is False
-
-
-def test_solve_log():
-    assert _solve_log(log(x**y) - y*log(x), x) == S.Zero
-    assert _solve_log(log(x)*log(y), x) is None
+def test_is_logarithm_reducable_to_single_instance():
+    assert _is_logarithm_reducable_to_single_instance(
+        Eq(log(x), 0), x) is False
+    assert _is_logarithm_reducable_to_single_instance(
+        Eq(log(x), -3), x) is False
+    assert _is_logarithm_reducable_to_single_instance(
+        Eq(log(x)*log(y), 0), x) is True
+    assert _is_logarithm_reducable_to_single_instance(
+        Eq(log(x)**2, x), 0) is False
+    assert _is_logarithm_reducable_to_single_instance(
+        Eq(log(x - 3) + log(x + 3), 0), x) is True
+    # assert _is_logarithm_reducable_to_single_instance(
+        # Eq(log(x**y) - y*log(x), 0), x) is True
```

Comment:
Commented out this test here and above, because I am not sure how do I handle such a scenario, where the equation gets reduced to a constant.
I was thinking if the arguments remain unevaluated when passed inside `logcombine`, but how this can be done? `evaluate=False` doesn't seems to work.